cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -204,6 +204,7 @@ def from_container(cls, container: dm.ContainerApply) -> "DMSContainer":
 class DMSView(SheetEntity):
     view: ViewType = Field(alias="View")
     implements: ViewListType | None = Field(None, alias="Implements")
+    in_model: bool = Field(True, alias="InModel")
```

Comment:
nice, this is what we discussed about stating certain views to be created but not added to the data model